pkg/cmd/init.go

```diff
@@ -34,6 +36,8 @@ func InitCommands(build, version string) *cobra.Command {
 	cmd.PersistentFlags().StringVar(&args.configPath, "config", "./runtime/config.yaml", "config file path")
 	cmd.PersistentFlags().StringVar(&args.logLevel, "log-level", "info", "log level (debug, info, warn, error)")
 	cmd.PersistentFlags().BoolVar(&args.textFormat, "log-text", false, "log in text format, otherwise JSON")
+	cmd.PersistentFlags().StringVar(&args.etcdURL, "etcdURL", "localhost:2379", "the host:port for etcd")
```

Comment:
maybe better to have separate `etcd.conf`

Reply:
@shan-96 lets use existing config file.